examples/vue-kitchen-sink/package.json

```diff
@@ -14,7 +14,7 @@
     "babel-loader": "^7.1.2",
     "babel-preset-env": "^1.6.0",
     "babel-preset-vue": "^1.2.1",
-    "cross-env": "^3.0.0",
+    "cross-env": "^5.1.1",
```

Comment:
Is it still needed here, given that we have it in root now?

Reply:
I wasn't sure. The kitchen sink is something people should be able to play with in isolation, right?

I assumed that, because `babel-core` and `babel-preset-env` were also duplicated, that the package `vue-kitchen-sink` should be self-contained.

Reply:
>  The kitchen sink is something people should be able to play with in isolation, right?

Not really. In our case, it‘s mainly used to test the changes made to other packages